docs/user_manual/source/control_status_registers.rst

```diff
@@ -1151,6 +1159,86 @@ Detailed:
 | 31:0  | RO       | Reads return 0x0, writes are ignored.                            |
 +-------+----------+------------------------------------------------------------------+
 
+
+CPU Control
+~~~~~~~~~~~
+CSR Address: 0xBF0
+
+Reset Value: 0x0000_0000
+
+Detailed:
+
++-------------+-----------+------------------------------------------------------------------------+
+|   Bit #     |   Mode    |   Description                                                          |
++=============+===========+========================================================================+
+| 19:16       | RW        | Frequency control for dummy instruction insertion.                     |
```

Comment:
Can you start the description fields with mentioning the bitfield names (rnddummyfreq, rnddata, etc.)?